app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
```

Comment:
There is a potential typo in the attribute name `hummer_level`. It might be intended to be `hammer_level` if referring to a tool or weapon. Consider renaming it for clarity.